compute/src/main/java/org/jclouds/compute/config/BaseComputeServiceContextModule.java

```diff
@@ -91,6 +92,7 @@ protected void configure() {
       }).to(CreateSshClientOncePortIsListeningOnNode.class);
       bind(new TypeLiteral<Function<TemplateOptions, Statement>>() {
       }).to(TemplateOptionsToStatement.class);
+      bind(TemplateBuilderImpl.class).to(ArbitraryCpuRamTemplateBuilderImpl.class);
```

Comment:
This cannot be done here. This binding must be applied in the module of each provider that supports an arbitrary cpu and ram config.